go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/google/gofuzz v1.0.0 // indirect
 	github.com/googleapis/gnostic v0.2.0 // indirect
 	github.com/gregjones/httpcache v0.0.0-20190212212710-3befbb6ad0cc // indirect
+	github.com/hashicorp/go-multierror v1.1.1 // indirect
```

Comment:
I am very hesitant to create new dependencies to 3rd party package like this. Can we not just concat errors? Is there anything special that this new package does that we can't easily do ourselves? These dependencies bring their own dependencies etc. We have tried to keep this repo lean.